src/components/shared/Card/Card.js

```diff
@@ -38,31 +39,60 @@ const Card = ({
   reportName,
   sourceUrl,
   itemStateName,
+  partiallyAvailable,
   children,
-}) => (
-  <div
-    className={cn("Card", className, {
-      "Card--population": isPopulation,
-      "Card--not-available": isNotAvailable || isTooStale,
-    })}
-  >
-    <div className="Card__header">
-      <h3 className="Card__title">{title}</h3>
-      {isTooStale && (
-        <div className="Card__warning-box Card__warning-box--warning">
-          <button type="button" tabIndex={0} className="Card__warning-icon" aria-label={hint} />
-          <div className="Card__warning">
-            <p>
-              This data is available in a public report from {itemStateName} (
-              <a className="Card__source-link" href={sourceUrl} target="_blank" rel="noreferrer">
-                {reportName}
-              </a>
-              ), but recent data is not available (Last Reported: {lastUpdatedDate}).
-            </p>
-            <br />
-            <p>
-              If you believe there is a public report containing this data, please let us know
-              at&nbsp;
+}) => {
+  const handleScroll = () => {
+    window.scrollTo({
+      top: 250,
+      behavior: "smooth",
+    });
+  };
+
+  return (
+    <div
+      className={cn("Card", className, {
+        "Card--population": isPopulation,
+        "Card--not-available": isNotAvailable || isTooStale,
+      })}
+    >
+      <div className="Card__header">
+        <h3 className="Card__title">{title}</h3>
+        {isTooStale && (
```

Comment:
In some cases, data is too stale to show on the "Monthly" tab, but we still show it on the annual tab. CA is an example of this:

<img width="469" alt="Screen Shot 2021-04-26 at 3 12 18 PM" src="https://user-images.githubusercontent.com/3762913/116158252-3096c780-a6a3-11eb-8abe-8d5e1134f08e.png">
<img width="424" alt="Screen Shot 2021-04-26 at 3 12 26 PM" src="https://user-images.githubusercontent.com/3762913/116158257-32608b00-a6a3-11eb-96dd-8fb63901ef3b.png">

This is pretty minor, but I think in this case it would be good to still show the "too stale" text, but then also add the "Use the reporting frequency switch" text so that they know they can see the stale data in the annual view. It is a bit confusing because it might not always be visible in the annual view, but I think it is better than it is currently. @hobuobi thoughts?

Reply:
Oh wait, so this is basically a case where both conditions are applicable, right? 
I think your suggestion is a good one, and I'm happy to go with it! In the future maybe we could make these error messages a bit more modular so that we could actually just show both? But not a thing to address right now, I think this is a small enough edge case